setup.cfg

```diff
@@ -73,6 +73,11 @@ jdaviz.configs.imviz.tests = data/*
 [options.entry_points]
 console_scripts =
     jdaviz = jdaviz.cli:_main
+    specviz = jdaviz.cli:_specviz
+    specviz2d = jdaviz.cli:_specviz2d
+    imviz = jdaviz.cli:_imviz
+    cubeviz = jdaviz.cli:_cubeviz
+    mosviz = jdaviz.cli:_mosviz
```

Comment:
awesome!  (Although maybe we should omit mosviz for now since we can't actually load anything directly from the UI afaik)

Reply:
Can you specify a directory via the file picker? I actually don't know 😅

Reply:
I think its planned, but currently unsupported

Reply:
AFAIK the current bundled file picker don't take dir as input but I am also not 100% sure. I agree with Kyle that filename should still be mandatory for Mosviz.

Reply:
Put in the filepath requirement for mosviz in https://github.com/spacetelescope/jdaviz/pull/1890/commits/e3a6d4b82ea23340b1b353aa862f5980ee095868